bookwyrm/templates/ostatus/template.html

```diff
@@ -0,0 +1,60 @@
+{% load layout %}
+{% load i18n %}
+{% load static %}
+{% load utilities %}
+{% load markdown %}
+
+<!DOCTYPE html>
+<html lang="{% get_lang %}">
+<head>
+    <title>{% block title %}{% endblock %}{{ site.name }}</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="{% static "css/vendor/bulma.min.css" %}">
+    <link rel="stylesheet" href="{% static "css/vendor/icons.css" %}">
+    <link rel="stylesheet" href="{% static "css/bookwyrm.css" %}">
+
+    <link rel="search" type="application/opensearchdescription+xml" href="{% url 'opensearch' %}" title="{% blocktrans with site_name=site.name %}{{ site_name }} search{% endblocktrans %}" />
+
+    <link rel="shortcut icon" type="image/x-icon" href="{% if site.favicon %}{% get_media_prefix %}{{ site.favicon }}{% else %}{% static "images/favicon.ico" %}{% endif %}">
+
+    {% if preview_images_enabled is True %}
+    <meta name="twitter:card" content="summary_large_image">
+    {% else %}
+    <meta name="twitter:card" content="summary">
+    {% endif %}
+    <meta name="twitter:title" content="{% if title %}{{ title }} - {% endif %}{{ site.name }}">
+    <meta name="og:title" content="{% if title %}{{ title }} - {% endif %}{{ site.name }}">
+    <meta name="twitter:description" content="{{ site.instance_tagline }}">
+    <meta name="og:description" content="{{ site.instance_tagline }}">
+
+    {% block opengraph_images %}
+        {% include 'snippets/opengraph_images.html' %}
+    {% endblock %}
+    <meta name="twitter:image:alt" content="BookWyrm Logo">
+    <script>
+        function closeWindow() {
+            window.close();
+        }
+    </script>
+</head>
+<body>
+<nav class="navbar" aria-label="main navigation">
+    <div class="container">
+        <div class="navbar-brand">
+            <a class="navbar-item" href="/">
```

Comment:
If we're in the pop-up window I think that all links could be disactivated, except for those necessary to the follow transaction